tests/test_ui.py

```diff
@@ -179,6 +196,24 @@ def addSample(self):
         widget_2 = self.getDockedWidget(self.window.docks, InsertPrimitiveDialog.dock_flag)
         self.assertIsNot(widget, widget_2)
         old_vertex_count = len(self.model.sample.vertices)
+
+        self.window.sample_properties_dialog_action.trigger()
+        sample_properties_widget = self.getDockedWidget(self.window.docks, SampleProperties.dock_flag)
+        self.assertTrue(isinstance(sample_properties_widget, SampleProperties))
+        bytes_used = self.model.sample.vertices.nbytes
+        bytes_to_mb_factor = 1 / (1024**2)
+        memory = bytes_used * bytes_to_mb_factor
+        memory_formatted = f'{memory:.4f}'
+        self.assertEqual('Memory (MB)', sample_properties_widget.sample_property_table.item(0, 0).text())
+        self.assertEqual(memory_formatted, sample_properties_widget.sample_property_table.item(0, 1).text())
+        num_indices, = self.model.sample.indices.shape
+        num_faces = num_indices // 3
+        self.assertEqual('Faces', sample_properties_widget.sample_property_table.item(1, 0).text())
+        self.assertEqual(str(num_faces), sample_properties_widget.sample_property_table.item(1, 1).text())
+        num_vertices, _ = self.model.sample.vertices.shape
```

Comment:
also unnecessary variable you can use `self.model.sample.indices.shape[0]` or if you keep the `num_vertices` above then there is no need to redefine it here